libs/blocks/notification/notification.js

```diff
@@ -135,6 +135,9 @@ async function decorateLockup(lockupArea, el) {
 async function decorateForegroundText(el, container) {
   const text = container?.querySelector('h1, h2, h3, h4, h5, h6, p')?.closest('div');
   text?.classList.add('text');
+  if (el.classList.contains('countdown-timer') && !el.matches('.pill, .ribbon')) {
+    await Promise.resolve(loadCDT(text, el.classList));
```

Comment:
This seems a bit odd, since you're not resolving based on some conditions. Wouldn't something like `await loadCDT(text, el.classList);` do the trick as well?